cmd/nerdctl/container_top_unix_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build linux || darwin || freebsd || netbsd || openbsd
```

Comment:
Could we do !windows?

Reply:
Probably you can just drop this line, as `unix` has been a built-in tag since Go 1.19

Reply:
Oh even better, didn't know that

Reply:
If the directive is not used, I get errors like this when running tests "cmd\nerdctl\container_top_test.go:38:62: undefined: testutil.AlpineImage".